README.md

```diff
@@ -1,4 +1,10 @@
-# Terraform Google Container VM Metadata Module
+# Terraform Google Container VM Metadata Module (DEPRECATED)
+
+> [!NOTE]
```

Comment:
I guess the intention is to have the whole note as a one block, but the way it looks right now - we have two sections.


<img width="1154" height="196" alt="Screenshot 2025-08-19 at 13 49 51" src="https://github.com/user-attachments/assets/66800622-3714-4979-b48a-9cba2f2d8e4e" />